src/pages/Profile/Tab/Tabs.jsx

```diff
@@ -0,0 +1,47 @@
+import { Wrapper, TabHead, EachTab,Outlet } from "./tabComponent";
+import {TabContent} from "./TabContent"
+import { useState } from "react";
+import { Post } from "../../../components";
+export const Tabs = () => {
+  const [activeTab, setActiveTab] = useState("tab1");
+  const currentStyle = {fontWeight: "700",color:"#ca2535",borderBottom:"3px solid #ca2535"}
+  return (
+    <Wrapper>
+      <TabHead>
+        <EachTab
+          style={activeTab === "tab1" ?currentStyle:null}
+          id="tab1"
+          onClick={(e) => setActiveTab(e.target.id)}
+        >
+          Posts
+        </EachTab>
+        <EachTab
+          style={activeTab === "tab2" ?currentStyle:null}
+          id="tab2"
+          onClick={(e) => setActiveTab(e.target.id)}
+        >
+          Liked
+        </EachTab>
+        <EachTab
+          style={activeTab === "tab3" ?currentStyle:null}
+          id="tab3"
```

Comment:
Why you need I don't get it

Reply:
ID*

Reply:
So here we have multiple tabs and id's are unique that's why I provided each tab with a id attribute and setting the `activeTab` the current tab's id on click event. We can also go with `name` attribute though